example/example-model/src/test/java/uk/gov/gchq/palisade/example/rule/BankDetailsTest.java

```diff
@@ -83,4 +87,21 @@ public void shouldRedactForNotPayrollAndNotSalary() {
         // Then
         assertNull(actual.getBankDetails());
     }
+
+    @Test
+    public void shouldDeserialiseExampleUser() {
```

Comment:
Please can this test be moved to BankDetailsRuleTest and then delete this file? They seem to be the same otherwise!